K2Bridge.Tests.UnitTests/Controllers/IndexListControllerTests.cs

```diff
@@ -38,6 +51,7 @@ private IndexListController GetController()
             var responseTask = Task.FromResult(response);
             mockDAL.Setup(kusto => kusto.GetFieldCapsAsync(It.IsNotNull<string>())).Returns(responseTask);
             mockDAL.Setup(kusto => kusto.GetIndexListAsync(It.IsNotNull<string>())).Returns(Task.FromResult(new IndexListResponseElement() { }));
+            mockDAL.Setup(kusto => kusto.ResolveIndexAsync(It.IsNotNull<string>())).Returns(Task.FromResult(new ResolveIndexResponse() { }));
```

Comment:
Isn't line 53 and its matching method (GetIndexListAsync) redundant?

Reply:
Removed.

Reply:
Why was this removed? do we know that kibana 7 doesn't request the index list in this way?

Reply:
Kibana 7 uses the `/_resolve/index` API and we did not want to keep the 6.x code around.

https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-resolve-index-api.html

Reply:
What about this comment from earlier -https://github.com/microsoft/K2Bridge/pull/62#discussion_r744722431?

Reply:
Kibana 7 does send a `/*%3A*/_search` request but it does not seem to be used. Index patterns management work fine without it.

Reply:
Below is a link to the source code of Kibana 7.10.2 where indices are searched for using `_search`:

https://github.com/elastic/kibana/blob/v7.10.2/src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/get_indices.ts#L199

According to the comment, "// CCS supports ±1 major version. We won't be able to expect resolve endpoint to exist until v9". 

So, my understanding is that the `_search` call is there only to supports Cross Cluster Search (CCS), which does not impact us.

Reply:
good discussion.